diff -up SOPE/sope-ldap/NGLdap/NGLdapConnection.m.omv~ SOPE/sope-ldap/NGLdap/NGLdapConnection.m
--- SOPE/sope-ldap/NGLdap/NGLdapConnection.m.omv~	2024-04-16 17:30:22.707359838 +0200
+++ SOPE/sope-ldap/NGLdap/NGLdapConnection.m	2024-04-16 17:31:49.112331521 +0200
@@ -704,7 +704,7 @@ static void freeMods(LDAPMod **mods) {
   rc = ldap_create_sort_control_value(self->handle, keys, &control_value);
   if (LDAP_SUCCESS == rc) {
     ctrls = malloc(2 * sizeof(LDAPControl));
-    rc = ldap_control_create(LDAP_CONTROL_SORTREQUEST, LDAP_OPT_ON, &control_value, 1, ctrls);
+    rc = ldap_control_create(LDAP_CONTROL_SORTREQUEST, true, &control_value, 1, ctrls);
     if (LDAP_SUCCESS == rc) {
       ctrls[1] = NULL;
       struct timeval tv_timelimit, *tv_timelimitp = NULL;
