diff -up SOPE/sope-gdl1/GDLAccess/EOEntity.m.omv~ SOPE/sope-gdl1/GDLAccess/EOEntity.m
--- SOPE/sope-gdl1/GDLAccess/EOEntity.m.omv~	2023-05-18 21:44:14.086111015 +0200
+++ SOPE/sope-gdl1/GDLAccess/EOEntity.m	2023-05-18 21:44:38.366341723 +0200
@@ -38,7 +38,7 @@
 #import <EOControl/EOKeyValueCoding.h>
 #import <EOControl/EOKeyGlobalID.h>
 
-static int _compareByName(id obj1, id obj2, void * context);
+static enum NSComparisonResult _compareByName(id obj1, id obj2, void * context);
 
 @interface NSObject(MappedArrayProtocol)
 - (NSArray *)mappedArrayUsingSelector:(SEL)_selector;
@@ -900,7 +900,7 @@ static inline BOOL _containsObject(NSArr
   return [dict count] > 0 ? dict : (NSMutableDictionary *)nil;
 }
 
-static int _compareByName(id obj1, id obj2, void * context) {
+static enum NSComparisonResult _compareByName(id obj1, id obj2, void * context) {
   return [[(EOAttribute*)obj1 name] compare:[(EOAttribute*)obj2 name]];
 }
 
