diff -up SOPE/sope-appserver/NGObjWeb/SoObjects/SoClassRegistry.m.omv~ SOPE/sope-appserver/NGObjWeb/SoObjects/SoClassRegistry.m
--- SOPE/sope-appserver/NGObjWeb/SoObjects/SoClassRegistry.m.omv~	2026-03-19 21:19:27.636137548 +0100
+++ SOPE/sope-appserver/NGObjWeb/SoObjects/SoClassRegistry.m	2026-03-19 21:20:03.148748202 +0100
@@ -126,7 +126,7 @@ static SoClassRegistry *registry = nil;
   
   if (_clazz == Nil)
     return nil;
-  if ((soClass = [self->objcToSoClass objectForKey:_clazz]))
+  if ((soClass = [self->objcToSoClass objectForKey:(id)_clazz]))
     return soClass;
   
   soSuper = [self soClassForClass:[_clazz superclass]];
@@ -136,7 +136,7 @@ static SoClassRegistry *registry = nil;
     [self debugWithFormat:@"could not create SoClass for class %@ !", _clazz];
     return nil;
   }
-  [self->objcToSoClass setObject:soClass forKey:_clazz];
+  [self->objcToSoClass setObject:soClass forKey:(id)_clazz];
   [soClass rescanClass];
   [self debugWithFormat:@"mapped class %@ to SoClass %@", 
 	  NSStringFromClass(_clazz), soClass];
diff -up SOPE/sope-appserver/NGObjWeb/WebDAV/SoSubscriptionManager.m.omv~ SOPE/sope-appserver/NGObjWeb/WebDAV/SoSubscriptionManager.m
--- SOPE/sope-appserver/NGObjWeb/WebDAV/SoSubscriptionManager.m.omv~	2026-03-19 21:21:18.077114564 +0100
+++ SOPE/sope-appserver/NGObjWeb/WebDAV/SoSubscriptionManager.m	2026-03-19 21:21:27.912927612 +0100
@@ -186,8 +186,8 @@ static SoSubscriptionManager *sm = nil;
     }
   }
   
-  if (pending  == nil) pending  = [NSArray array];
-  if (inactive == nil) inactive = [NSArray array];
+  if (pending  == nil) pending  = [NSMutableArray array];
+  if (inactive == nil) inactive = [NSMutableArray array];
   return [NSDictionary dictionaryWithObjectsAndKeys:
                          pending,  @"pending",
                          inactive, @"inactive",
diff -up SOPE/sope-appserver/WEExtensions/WETableView/WETableView.m.omv~ SOPE/sope-appserver/WEExtensions/WETableView/WETableView.m
--- SOPE/sope-appserver/WEExtensions/WETableView/WETableView.m.omv~	2026-03-19 21:21:58.364784758 +0100
+++ SOPE/sope-appserver/WEExtensions/WETableView/WETableView.m	2026-03-19 21:22:04.210196027 +0100
@@ -462,7 +462,7 @@ static inline void _applyState_(WETableV
     infos = [_ctx objectForKey:WETableView_INFOS];
 
     if (infos == nil)
-      infos = [NSArray array];
+      infos = [NSMutableArray array];
 
     NSAssert(infos != nil, @"Infos is nil.");
 
diff -up SOPE/sope-gdl1/MySQL/MySQL4Values.m.omv~ SOPE/sope-gdl1/MySQL/MySQL4Values.m
--- SOPE/sope-gdl1/MySQL/MySQL4Values.m.omv~	2026-03-19 21:23:25.394034528 +0100
+++ SOPE/sope-gdl1/MySQL/MySQL4Values.m	2026-03-19 21:23:32.451374021 +0100
@@ -41,7 +41,7 @@
   typeName = _dt;
 
   if (typeName == nil)
-    typeName = [NSString stringWithFormat:@"Oid[%i]", _dt];
+    typeName = [NSString stringWithFormat:@"Oid[%@]", _dt];
 
   r = [NSString stringWithFormat:
                               @"mapping between %@<Class:%@> and "
diff -up SOPE/sope-mime/NGImap4/NGImap4Folder.m.omv~ SOPE/sope-mime/NGImap4/NGImap4Folder.m
--- SOPE/sope-mime/NGImap4/NGImap4Folder.m.omv~	2026-03-19 21:12:09.674218584 +0100
+++ SOPE/sope-mime/NGImap4/NGImap4Folder.m	2026-03-19 21:12:24.283034167 +0100
@@ -300,7 +300,7 @@ static int FetchNewUnseenMessagesInSubFo
   msn  = [[dict objectForKey:@"search"] mutableCopy];
   
   if ((msn == nil) || ![msn isNotEmpty]) {
-    mes = [NSArray array];
+    mes = [NSMutableArray array];
   }
   else {
     NSEnumerator *seq = nil;
diff -up SOPE/sope-mime/NGImap4/NGImap4ResponseParser.m.omv~ SOPE/sope-mime/NGImap4/NGImap4ResponseParser.m
--- SOPE/sope-mime/NGImap4/NGImap4ResponseParser.m.omv~	2026-03-19 21:13:43.066198201 +0100
+++ SOPE/sope-mime/NGImap4/NGImap4ResponseParser.m	2026-03-19 21:16:17.120733137 +0100
@@ -88,7 +88,7 @@ static NSDictionary *_parseSingleBody(NG
 static NSDictionary *_parseMultipartBody(NGImap4ResponseParser *self,
 					 BOOL isBodyStructure);
 
-static NSArray *_parseLanguages();
+static NSArray *_parseLanguages(NGImap4ResponseParser *self);
 
 static NSString *_parseBodyString(NGImap4ResponseParser *self,
                                   BOOL _convertString);
@@ -97,7 +97,7 @@ static NSString *_parseBodyDecodeString(
                                         BOOL _decode);
 static NSDictionary *_parseBodyParameterList(NGImap4ResponseParser *self);
 static NSDictionary *_parseContentDisposition(NGImap4ResponseParser *self);
-static NSArray *_parseAddressStructure(NGImap4ResponseParser *self);
+static NSDictionary *_parseAddressStructure(NGImap4ResponseParser *self);
 static NSArray *_parseParenthesizedAddressList(NGImap4ResponseParser *self);
 static int _parseTaggedResponse(NGImap4ResponseParser *self,
                                 NGMutableHashMap *result_);
@@ -2205,7 +2205,7 @@ static NSDictionary *_parseContentDispos
   return disposition;
 }
 
-static NSArray *_parseAddressStructure(NGImap4ResponseParser *self) {
+static NSDictionary *_parseAddressStructure(NGImap4ResponseParser *self) {
   NSString *personalName, *sourceRoute, *mailboxName, *hostName;
 
   // Handle broken address list - generally sent by Microsoft Exchange.
