diff -up SOPE/sope-mime/NGMime/NGMimeType.m.omv~ SOPE/sope-mime/NGMime/NGMimeType.m
--- SOPE/sope-mime/NGMime/NGMimeType.m.omv~	2023-01-19 06:10:04.547084118 +0100
+++ SOPE/sope-mime/NGMime/NGMimeType.m	2023-01-19 06:10:19.587166284 +0100
@@ -125,14 +125,11 @@ static Class NSStringClass  = Nil;
     encoding = NSKoreanEUCStringEncoding;
   }
   else if ([charset isEqualToString:@"big5"]) {
-    encoding = NSBIG5StringEncoding;
+    encoding = NSBig5StringEncoding;
   }
   else if ([charset isEqualToString:@"iso-2022-jp"]) {
     encoding = NSISO2022JPStringEncoding;
   }
-  else if ([charset isEqualToString:@"gb2312"]) {
-    encoding = NSGB2312StringEncoding;
-  }
   else if ([charset isEqualToString:@"koi8-r"]) {
     encoding = NSKOI8RStringEncoding;
   }
